docs/docs/tutorial/chapter1/prerequisites.md

```diff
@@ -47,16 +47,17 @@ Please do upgrade accordingly. Then proceed to the Redwood installation when you
 There are many ways to install and manage both Node.js and Yarn. If you're installing for the first time, we recommend the following:
 
 **1. Yarn**
-We recommend following the [instructions via Yarnpkg.com](https://classic.yarnpkg.com/en/docs/install/).
+We recommend following the [instructions via Yarnpkg.com](https://yarnpkg.com/getting-started/install).
 
 **2. Node.js**
-Using the recommended [LTS version from Nodejs.org](https://nodejs.org/en/) is preferred, as the latest Current version isn't supported.
+Using the recommended [LTS version from Nodejs.org](https://nodejs.org/en/) is preferred.
```

Comment:
I don't want people to get the idea that they can't use v20. They can, they just can't deploy it. It's true we don't run CI on v20, but we've gotten in too much hot water before for "not supporting" the latest Node version. It's not like any of our packages are fundamentally incompatible with the latest Node version, we just don't recommend it. I say let them do what they want after saying the latest LTS is preferred